chemprop/v2/tests/unit/featurizers/test_bond.py

```diff
@@ -65,9 +65,8 @@ def test_bt_bit(x, bt_bit):
     assert x[bt_bit] == 1
 
 
-def test_conj_bit(featurizer, x, bond):
-    conj_bit = featurizer.subfeatures["conjugated"].start
-
+def test_conj_bit(featurizer, bond, bond_types, x):
+    conj_bit = featurizer.one_hot_index(bond_types, featurizer.bond_types)[1] 
     assert x[conj_bit] == int(bond.GetIsConjugated())
```

Comment:
While the results stay the same, I think, instead of introducing `bond_types` as an argument and using `one_hot_index`, It is more proper to do `conj_bit = 1 + len(featurizer.bond_types)`.

Reply:
I have changed this accordingly.